cosmosdb/client_endpoint.bal

```diff
@@ -37,321 +38,232 @@ public isolated client class DataPlaneClient {
     #
     # + cosmosdbConfig - Configurations required to initialize the `Client` endpoint
     # + httpClientConfig - HTTP configuration
-    # + return -  Error at failure of client initialization
-    public isolated function init(ConnectionConfig cosmosdbConfig, http:ClientConfiguration httpClientConfig = {}) returns Error? {
+    # + return - Error at failure of client initialization
+    public isolated function init(ConnectionConfig cosmosdbConfig, http:ClientConfiguration httpClientConfig = {},
+    CustomizedConfiguration? customConfig = ()) returns Error? {
```

Comment:
Shall we remove ClientConfiguration as we are using the SDK?

Reply:
cosmosdbConfig --> connectionConfig
customConfig --> advanceClientConfig (Type needs to be ClientConfiguration)